engine/engine/management/commands/remove_field.py

```diff
@@ -0,0 +1,64 @@
+from django.core.management import BaseCommand
+from django.db import connection
+from django.db.migrations import Migration
+from django.db.migrations.autodetector import MigrationAutodetector
+from django.db.migrations.loader import MigrationLoader
+from django.db.migrations.writer import MigrationWriter
+
+from common.migrations.remove_field import RemoveFieldDB, RemoveFieldState
+
+
+class Command(BaseCommand):
```

Comment:
Example: `python manage.py remove_field alerts AlertReceiveChannel restricted_at` (assuming that field exists)

This generates two migrations:

`0031_remove_alertreceivechannel_restricted_at_state.py`

```python
# Generated by Django 3.2.20 on 2023-08-07 20:26

import common.migrations.remove_field
from django.db import migrations


class Migration(migrations.Migration):

    dependencies = [
        ('alerts', '0030_auto_20230731_0341'),
    ]

    operations = [
        common.migrations.remove_field.RemoveFieldState(
            model_name='AlertReceiveChannel',
            name='restricted_at',
        ),
    ]
```

`0032_remove_alertreceivechannel_restricted_at_db.py`

```python
# Generated by Django 3.2.20 on 2023-08-07 20:26

import common.migrations.remove_field
from django.db import migrations


class Migration(migrations.Migration):

    dependencies = [
        ('alerts', '0031_alertreceivechannel_restricted_at'),
    ]

    operations = [
        common.migrations.remove_field.RemoveFieldDB(
            model_name='AlertReceiveChannel',
            name='restricted_at',
            remove_state_migration=('alerts', '0031_remove_alertreceivechannel_restricted_at_state'),
        ),
    ]
```